xbmc/pvr/PVRPlaybackState.h

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include "addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr.h"
```

Comment:
Including `pvr_general.h` should be sufficient. `pvr.h` includes everything from the c-API and you only need `PVR_SOURCE`here.

Reply:
Done